receiver/azureeventhubreceiver/client.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package azureeventhubreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azureeventhubreceiver"
+package azureeventhubreceiver // import "github.com/loomis-relativity/opentelemetry-collector-contrib/receiver/azureeventhubreceiver"
```

Comment:
```suggestion
package azureeventhubreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azureeventhubreceiver"
```

Reply:
@atoulme This was changed just to allow me to pull this into our customized collector easily for testing. It'll be changed back to the standard module name before I take the PR out of draft status.

Reply:
OK!

Reply:
Bump

Reply:
Done.